docker/cortx-deploy/Dockerfile

```diff
@@ -61,18 +61,18 @@ RUN yum install yum-utils -y \
 COPY ./*.txt ./${CORTX_IMAGE}/*.txt ./compatibility.info ./generate-release-info.sh /opt/seagate/cortx/
 
 RUN yum install --nogpgcheck -y python3-pip \
-    && pip3 install yq \
+    && pip3 install yq==2.14.0 \
```

Comment:
@shailesh-vaidya, should we use below yq version here as well?
https://github.com/Seagate/cortx-re/blob/3bf8fdcf0eda4cc2a3829d6b918013b89cd0220c/solutions/kubernetes/cortx-deploy-functions.sh#L27
CC - @subhalaxmisa

Reply:
@gauravchaudhari02 - YQ_VERSION=v4.25.1  is not available through pip. And anyways we are planning to remove `yq `from an image. So let this as it is.